fulp_modules/features/jobs/deputy/deputy.dm

```diff
@@ -37,15 +37,21 @@
 		/obj/item/choice_beacon/music = 5,
 		/obj/item/crowbar/large = 1,
 		/obj/item/melee/baton/security/boomerang/loaded = 1,
-		/obj/item/clothing/gloves/tackler/rocket = 1,
```

Comment:
not mentioned in the changelog, smh my head my head

Reply:
No fun allowed. I'd really prefer if you keep this there.

Reply:
DEPUTIES ARENT MEANT TO TACKLE THEY ARENT MEANT TO GET ANY TACKLING GLOVES TH